src/main/java/com/angelbroking/smartapi/utils/Utils.java

```diff
@@ -5,11 +5,11 @@ private Utils() {
 		
 	}
 
-	public static boolean isEmpty(final CharSequence cs) {
-		return cs == null || cs.length() == 0;
+	public static boolean isEmpty(final Integer nm) {
+		return nm == null || nm.equals(0);
 	}
 
-	public static boolean isNotEmpty(final CharSequence cs) {
+	public static boolean isNotEmpty(final Integer cs) {
```

Comment:
No need for this method. Unnecessary duplicating code.

Reply:
it is removed now.